Sources/Networking/AlamofireNetworkClient/AlamofireConsoleLogger.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  AlamofireConsoleLogger.swift
+//  PovioKit
+//
+//  Created by Borut Tomažin on 05/11/2020.
+//  Copyright © 2020 Povio Labs. All rights reserved.
+//
+
+import Alamofire
+import Foundation
+
+public final class AlamofireConsoleLogger: EventMonitor {
+  public let queue = DispatchQueue(label: "com.alamofire.console.networklogger")
+  private let logger: Writer
+  
+  public init(logger: @escaping Writer = { PovioKit.Logger.debug($0) }) {
+    self.logger = logger
+  }
+  
+  public func request(_ request: Request, didCreateInitialURLRequest urlRequest: URLRequest) {
+    logger("\(request.method) (start) to \(request.endpoint).")
+  }
+  
+  public func requestDidFinish(_ request: Request) {
+    logger("\(request.method) (success: \(request.statusCode)) to \(request.endpoint)")
+  }
+  
+  public func request(_ request: Request, didFailTask task: URLSessionTask, earlyWithError error: AFError) {
+    logger("\(request.method) (fail: \(request.statusCode)) to \(request.endpoint) due to: \(error.localizedDescription)")
+  }
+}
+
+// MARK: - Private Getters
+private extension Request {
+  var method: String {
+    request?.method?.rawValue ?? "<no method>"
+  }
+  
+  var statusCode: Int {
+    response?.statusCode ?? 0
+  }
+  
+  var endpoint: String {
+    request?.url?.lastPathComponent ?? "<no endpoint>"
```

Comment:
Hm, might be handy to return everything after host - just in case we're in a situation where we have v1/user, v2/user endpoints (*sigh*)

That way we can skim the logs and see they're 2 different endpoints

Reply:
That's a great suggestion, smth we should definitely do.

Reply:
@nejcvivod https://github.com/poviolabs/PovioKit/pull/83/commits/c5dc4a25c6751a51cbc0e783a7b48bd5905ec9e1